gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -251,6 +251,14 @@ nnstreamer_query_receive (query_connection_handle connection,
           return -EREMOTEIO;
         }
         return 0;
+      } else if (data->cmd == _TENSOR_QUERY_CMD_CLIENT_ID) {
+        /* receive client id */
+        data->client_id = g_malloc0 (37);
```

Comment:
How about using a ```const``` variable instead of 37.

Reply:
Thanks. Updated.